index.html

```diff
@@ -0,0 +1,37 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Portfolio: setup and mobile first</title>
+  <link rel="stylesheet" href="style.css">
+  <link href='https://fonts.googleapis.com/css?family=Roboto' rel='stylesheet'>
+  <link href='https://fonts.googleapis.com/css?family=Crete Round' rel='stylesheet'>
+  <link href='https://fonts.googleapis.com/css?family=Poppins' rel='stylesheet'>
+</head>
+<body>
+  <main>
+    <header>
+      <div class="icon">
+        <time class="time">9:41</time>
+        <div class="bar"></div>  
+      </div>
+
+      <nav class='toolbar'>
+        <a class="logo" href="#"></a>
+        <a class="menu" href="#"></a>
+      </nav>
+    </header>
+
+    <section class="headline">
+      <h1>Hey There. <br> I'm Anita</h1>
+      <h2>I am a Software Developer</h2>
+      <p>I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, do not hestiate to contact me.</p>
+      <a class="social_media"></a>
```

Comment:
- [ ] to be semantically correct please note that the social media should be separate links and icons, instead of and background image, kindly add wrap your social links a ul tag to group them, and li per icon; each icon has a link (a tag) 🙏

Reply:
Done!